Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:jessie
```

Comment:
I'm not super familiar with dockerfiles anymore - is this something specific to your machine? (specifically `jessie`)

Reply:
No, it's [the debian version](https://www.debian.org/releases/jessie/). I guess this should work with any distro that runs `apt`, but this variant is the only one I tried.